view/template/developer/_quickstartCredits.php

```diff
@@ -19,7 +19,16 @@
   "name": "electricsheep",
   "file_path": "\\home\kauffj\\Desktop\\electric-sheep.mp4",
   "bid": 1,
-  "metadata":  { } <span class="code-bash__comment">//this should match the metadata returned by resolve_name </span>
+  "metadata":  {
+    "description": "Some interesting content",
+    "title": "my_interesting_content",
+    "author": "Video shot by me@example.com",
+    "language": "en",
```

Comment:
For language, would there be any restriction on what kind of values should be entered? For instance [ISO-639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). Otherwise there would be content with `"language": "en" ` and `"language": "English"`, etc, which would make filtering more difficult.

Also, what about some sort of null or not-applicable value? In my case, a photograph of a bird doesn't have a language.